_commands/basics/printf.md

```diff
@@ -0,0 +1,99 @@
+---
+---
+
+printf
+--
+
+`printf` is used to format and output data.
+
+~~~ bash
+$ printf format [argument]
+$ printf "%f\n" 7
+7.000000
+$ printf "%d\n " 0xF
+15
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `printf`
+~~~ bash
+$ printf "%f\n" 5
+~~~
+
+##### Break it down
+
+*This is a simple use of 'printf'. It prints out the number 5 as a float on a new line. The default is to print with 6 decimals.
```

Comment:
>*This is a simple use of 'printf'

can be removed.
You could instead modify the next sentence and start it with "This example prints out the number 5..."